src/System.IO.FileSystem.Primitives/tests/System.IO.FileSystem.Primitives.Tests.builds

```diff
@@ -4,7 +4,11 @@
   <ItemGroup>
     <Project Include="System.IO.FileSystem.Primitives.Tests.csproj"/>
     <Project Include="System.IO.FileSystem.Primitives.Tests.csproj">
-      <TestTFMs>netcore50;net46</TestTFMs>
```

Comment:
why did you remove the netcore50 test configuration? I would expect we should still make sure we test that configuration.